scripts/visualise_results.py

```diff
@@ -0,0 +1,76 @@
+import sys
+from glob import glob
+
+import statistics
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def get_stats(folders):
+    results_list = []
+
+    for folder in folders:
+
+        get_request_response_times = []
+        post_request_response_times = []
+        all_response_times = []
+
+        for file in glob(folder + '/*distribution.csv'):
+
+            with open(file) as f:
+                data = f.read()
+
+            get_values = []
+            post_values = []
+
+            for line in data.split('\n'):
+                if 'Name' in line:
+                    continue
+
+                values = line.split(',')
+                percentile_99th = int(values[9])
+
+                if 'GET /questionnaire' in line:
+                    get_values.append(percentile_99th)
+                if 'POST /questionnaire' in line:
+                    post_values.append(percentile_99th)
+
+            get_request_response_times.extend(get_values)
+            post_request_response_times.extend(post_values)
+
+            all_response_times = get_values + post_values
+
+        date = folder.split('T')[0].split('/')[-1]
```

Comment:
The split on `/` should be first, so it reads as "get the parent folder name and split on the date".